com.unity.template.renderstreaming-hd/Assets/Scripts/RenderStreaming.cs

```diff
@@ -71,6 +72,20 @@ enum UnityEventType
 
         public void Awake()
         {
+            if (Application.platform != RuntimePlatform.LinuxEditor && Application.platform != RuntimePlatform.LinuxPlayer)
```

Comment:
I guess this check is better to put to `WebRTC.Initialize` than here.

Reply:
okey, I move to this check `WebRTC.Initialize`